### PR TITLE
Update token scope api call version

### PR DESCRIPTION
**Pull request by 0sewa0**
## Description

We use the older v1 tenant API for token scope check, this APIs are now deprecated.
However this APIs didn't change in the v2 tenant API, so we only need to change the URL we call, and nothing more.

## How can this be tested?

no change in behaviour (token scope check)

### CODE REVIEW

**Review by waodim — APPROVED** [review]

**Review by albertogdd — APPROVED** [review]
Deployed branch in my cluster, LGTM

**Review by chrismuellner — CHANGES_REQUESTED** [review]
doesn't work

File: pkg/clients/dynatrace/dynatrace_client_test.go [hunk 1]
```diff
@@ -174,9 +174,9 @@ func handleRequest(request *http.Request, writer http.ResponseWriter) {
 		(&ipHandler{}).ServeHTTP(writer, request)
 	case "/v1/deployment/installer/agent/connectioninfo":
 		handleCommunicationHosts(request, writer)
-	case "/v1/events":
+	case "/v2/events":
```
Comment (chrismuellner) [inline pkg/clients/dynatrace/dynatrace_client_test.go:177]: this endpoint does not exist in v2 as post:
- `/v1/events` to list/post events
- `/v2/events` to list all events
- `/v2/events/ingest` to ingest a custom event

**Review by chrismuellner — CHANGES_REQUESTED** [review]

File: pkg/clients/dynatrace/dynatrace_client_test.go (hunk 1 above)
Comment (0sewa0) [reply pkg/clients/dynatrace/dynatrace_client_test.go]: thankfully we only POST, so we don't have to care about `/v2/events` 😌 

(quickly added it, still need to test that it actually works)

Comment (0sewa0) [reply pkg/clients/dynatrace/dynatrace_client_test.go]: removed it as it requires a different API scope

**Review by chrismuellner — APPROVED** [review]

**Review by aorcholski — APPROVED** [review]
```
{"level":"info","ts":"2024-07-25T14:49:22.240Z","logger":"dynatrace-client","msg":"token verified"}
```

**Pull request merged**